providers/ringcentral/src/lib/ringcentral.provider.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+
+const RC = require('@ringcentral/sdk').SDK;
+require('dotenv').config();
```

Comment:
Unfortuently due to the multi-tenancy of the system we can't use env variables in the provider. All configuration variables need to be passed from the constructor.